armi/reactor/reactors.py

```diff
@@ -618,7 +618,7 @@ def getNumAssembliesWithAllRingsFilledOut(self, nRings):
             return nRings * (nRings - 1) + (nRings + 1) // 2
 
     def getNumEnergyGroups(self):
-        """"
+        """ "
```

Comment:
extra "

Reply:
huh. good old new black version?

Reply:
an extra quote is an extra quote. my guess is black is hesitant to actually remove it